### PR TITLE
Endpoint to follow multiple spotify artists

### PR DESCRIPTION
**Pull request by hedgehog1833**
A new endpoint to follow multiple spotify artists. This endpoint fetched all artists from Spotify that are not yet in our database and then saves all FollowActionEntities for the current user.

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[Artist import endpoints](https://trello.com/c/ILFpbOQk/243-artist-import-endpoints)

File: webapp/src/main/java/rocks/metaldetector/service/artist/FollowArtistServiceImpl.java [hunk 1]
```diff
@@ -104,6 +120,13 @@ private ArtistEntity saveAndFetchArtist(String externalId, ArtistSource source)
     return artistRepository.save(artistEntity);
   }
 
+  private List<ArtistEntity> saveAndFetchArtists(List<String> spotifyArtistIds) {
+    List<String> newArtistsIds = artistService.findNewArtistIds(spotifyArtistIds);
+    List<SpotifyArtistDto> spotifyArtistDtos = spotifyService.searchArtistsByIds(newArtistsIds);
+    artistService.persistArtists(spotifyArtistDtos);
+    return artistRepository.findAllByExternalIdIn(spotifyArtistIds);
```
Comment (DanielW1987) [inline webapp/src/main/java/rocks/metaldetector/service/artist/FollowArtistServiceImpl.java:127]: This is somewhat redundant.

Via `artistService.persistArtists(spotifyArtistDtos)` the the method `artistRepository.saveAll(artistEntities)` is called and this already returns all ArtistEntities.

Comment (hedgehog1833) [reply webapp/src/main/java/rocks/metaldetector/service/artist/FollowArtistServiceImpl.java:127]: No :) Only those artists are persisted, that we don't already have in our database. But what we need are all artists that were requested. So we need this call here.

Comment (DanielW1987) [reply webapp/src/main/java/rocks/metaldetector/service/artist/FollowArtistServiceImpl.java:127]: maybe you can name `spotifyArtistDtos` to `newSpotifyArtistDtos` to make it less confusing.

**Review by DanielW1987 — APPROVED** [review]

**Pull request merged**